loops.js

```diff
@@ -50,4 +63,14 @@
 // >> 8
 // >> 9
 // ==========================================
-
```

Comment:
Dit is het antwoord van NOVI, persoonlijk vind ik jou antwoord beter ;)

`for (let i = 0; i < 10; i++) {
  if (i < 3) {
    console.log(i);
  }
  if (i >= 3 && i < 6) {
    console.log('> ' + i);
  }
  if (i >= 6) {
    console.log('>> ' + i);
  }
}`